templates/selenium/0/docker-compose.yml.tpl

```diff
@@ -0,0 +1,28 @@
+version: '2'
+services:
+  {{- if eq .Values.DEPLOY_SELENIUM_HUB "true"}}
+  seleniumhub:
+    image: selenium/hub:${SELENIUM_HUB_VERSION}
+    environment:
+      GRID_TIMEOUT: ${GRID_TIMEOUT}
+    ports:
+      - ${PUBLISH_PORT}:4444
+    labels:
+      io.rancher.container.pull_image: always
+  {{- end}}
+  {{- if eq .Values.DEPLOY_SELENIUM_CHROME "true"}}
+  selenium-chrome:
+    image: selenium/node-chrome:${SELENIUM_CHROME_VERSION}
+    links:
+    - seleniumhub:hub
+    labels:
+      io.rancher.container.pull_image: always
+  {{- end}}
+  {{- if eq .Values.DEPLOY_SELENIUM_FIREFOX "true"}}
+  selenium-firefox:
+    image: selenium/node-firefox:${SELENIUM_FIREFOX_VERSION}
+    links:
+    - seleniumhub:hub
+    labels:
+      io.rancher.container.pull_image: always
+  {{- end}}
```

Comment:
If `DEPLOY_SELENIUM_HUB`, `DEPLOY_SELENIUM_CHROME` and `DEPLOY_SELENIUM_FIREFOX` set to false, nothing to deploy??
As commented below, may be `DEPLOY_SELENIUM_HUB` is not needed??

Reply:
I have made hub as mandatory